BostonData/lambda_function/lambda_function.py

```diff
@@ -116,9 +116,12 @@ def get_welcome_response():
 
     session_attributes = {}
     card_title = "Welcome"
-    speech_output = "Welcome to the Boston Data skill. " \
-                    "To set your address, say " \
-                    "my address is, followed by your address."
+    speech_output = "Welcome to the Boston Public Services skill. " \

```

Comment:
I like the new name! If others like it too we should change this in the Readme as the new skill name and invocation name.